nixos/doc/manual/installation/installing-from-other-distro.section.md

```diff
@@ -231,10 +237,45 @@ The first steps to all these are the same:
     $ echo etc/nixos | sudo tee -a /etc/NIXOS_LUSTRATE
     ```
 
-1.  Finally, move the `/boot` directory of your current distribution out
-    of the way (the lustrate process will take care of the rest once you
-    reboot, but this one must be moved out now because NixOS needs to
-    install its own boot files:
+1.  Finally, install NixOS's boot system, backing up the existing boot
+    system's files in the process.
+
+    The details of this step can vary depending on the bootloader
+    configured in NixOS and the bootloader in use by the current
+    system.
+
+    The commands below should work for:
+
+    - BIOS systems.
+
+    - UEFI systems where both the current system and NixOS mount the
+      EFI System Partition on `/boot`, which both systemd-boot and
```

Comment:
Please link to (at least somewhat) authoritative definitions of those technical terms; not every reader who may need these instructions will be familiar with them. Wikipedia is alright for me, ideally it would be the canonical upstream, but it should be something at all.

- UEFI
- ESP
- systemd-boot